tests/jupyter_notebook_config.py

```diff
@@ -0,0 +1,19 @@
+# Configuration file for jupyter-notebook.
+
+from tempfile import mkdtemp
+
+
+c = get_config()  #noqa
```

Comment:
What is this?

Reply:
I got this from calling `jupyter notebook --generate-config` because I was running into issues with the config not being picked up by the notebook server.